src/openrct2/rct1.h

```diff
@@ -30,6 +30,12 @@
 #define RCT1_MAX_MAP_ELEMENTS       0xC000
 #define RCT1_MAX_SPRITES            5000
 #define RCT1_MAX_VEHICLES_PER_RIDE  12
+#define RCT1_MAX_STATIONS           4
+#define RCT1_MAX_TRAINS_PER_RIDE    12
```

Comment:
Could you change existing occurrences of RCT1_MAX_VEHICLES_PER_RIDE to RCT1_MAX_TRAINS_PER_RIDE and remove RCT1_MAX_VEHICLES_PER_RIDE?